nengo/config.py

```diff
@@ -139,16 +133,13 @@ def __init__(self, configures, clsparams):
     def __contains__(self, key):
         return self in self._clsparams.get_param(key)
 
-    def __getattribute__(self, key):
-        try:
-            return super(InstanceParams, self).__getattribute__(key)
-        except AttributeError:
-            if key in self._clsparams.default_params:
-                raise
-            param = self._clsparams.get_param(key)
-            if self in param:
-                return param.__get__(self, self.__class__)
-            return getattr(self._clsparams, key)
+    def __getattr__(self, key):
+        if key in self._clsparams.default_params:
+            raise AttributeError()
```

Comment:
This is an error because we don't want there to be two separate and non-interacting ways to set an object's attributes. Using radius as an example, if this wasn't an error, then these would both be possible:
```python
ens.radius = 2.0
net.config[ens].radius = 3.0
```
You might think that these lines both change the ensemble's radius, but they do not; the second line won't have any effect. We raise this error so that the second line is not possible.